src/contexts/AuthContext.tsx

```diff
@@ -1,14 +1,17 @@
 import React, { createContext, useCallback, useEffect, useState } from 'react';
 import moment from 'moment';
 
-import { IAuthParams, IUser, IUserWithExpiration } from '../types';
+import { IAuthParams, IUser, IUserWithExpiration, IResetPasswordParams } from '../types';
 import { UserAPI } from '../api';
 
 export interface IAuthContext {
   isAuthenticated: boolean;
   authenticating: boolean;
+  isPasswordResetted: boolean;
```

Comment:
there's no need for this actually, you can handle everything inside the `ResetPassword` component